ui/src/buckets/actions/thunks.ts

```diff
@@ -51,6 +51,8 @@ import {
   removeBucketLabelFailed,
 } from 'src/shared/copy/notifications'
 import {LIMIT} from 'src/resources/constants'
+import {getBucketsLabels} from 'src/client'
```

Comment:
```suggestion
```

Reply:
this gets used on line 104 to get the labels for the bucket. is there a better way to make this api call?

Reply:
never mind, I just saw your comment below where I can do api.getBucketsLabels. thank you!